internal/providers/system/system.go

```diff
@@ -30,12 +31,24 @@ import (
 )
 
 const (
-	baseFilename = "base.ign"
 	userFilename = "user.ign"
 )
 
-func FetchBaseConfig(logger *log.Logger) (types.Config, report.Report, error) {
-	return fetchConfig(logger, baseFilename)
+func FetchBaseConfig(logger *log.Logger, platformName string) (types.Config, report.Report, error) {
```

Comment:
Would be nice to add a doc string to what is happening here (and the expected/checked paths).